summingbird-core-test/src/main/scala/com/twitter/summingbird/TestPlatform.scala

```diff
@@ -0,0 +1,41 @@
+package com.twitter.summingbird
+
+import com.twitter.algebird.Semigroup
+import org.scalacheck.Arbitrary
+
+class TestPlatform extends Platform[TestPlatform] {
+  override type Source[T] = TestPlatform.SourceList[T]
+  override type Store[K, V] = TestPlatform.Store[K, V]
+  override type Sink[T] = TestPlatform.Sink[T]
+  override type Service[K, V] = TestPlatform.Service[K, V]
+  override type Plan[T] = TailProducer[TestPlatform, T]
+
+  override def plan[T](completed: TailProducer[TestPlatform, T]): TailProducer[TestPlatform, T] = completed
+}
+
+object TestPlatform {
+  // Test `Source` should accept data with time, not just plain data.
+  case class SourceList[T](data: List[T])
+  case class Store[K, V: Semigroup](id: String, initial: Map[K, V]) {
+    val valueSemigroup: Semigroup[V] = implicitly[Semigroup[V]]
+  }
+  case class Sink[T: Ordering](id: String) {
+    val ordering: Ordering[T] = implicitly[Ordering[T]]
```

Comment:
Can you comment here why ordering is needed for the Test Platform? As I understand it is because one of the other platforms requires it and without this we won't be able to transform. It's not very obvious though, so a comment would be useful.